cmd/lilypad/solver.go

```diff
@@ -20,6 +21,8 @@ func newSolverCmd() *cobra.Command {
 		Long:    "Start the lilypad solver service.",
 		Example: "",
 		RunE: func(cmd *cobra.Command, _ []string) error {
+			optionsfactory.CheckDeprecation(data.ServiceConfig{}, options.Web3)
```

Comment:
The solver does not have a service config.

Reply:
It doesn't? Then we should remove the `SERVICE_MEDIATORS` key/value pair from the `solver`'s environments in doppler.

Reply:
Yep agreed. To back this up, the solver options are here:

https://github.com/Lilypad-Tech/lilypad/blob/d5f16cadbed8f9b8223192357f9ccf9eb068d522/pkg/options/solver.go#L9-L16

The commands that configure services call `GetDefaultServicesOptions`:

https://github.com/Lilypad-Tech/lilypad/blob/d5f16cadbed8f9b8223192357f9ccf9eb068d522/pkg/options/services.go#L10-L15